libs/feedback/src/lib/feedback-widget/feedback-widget.component.ts

```diff
@@ -10,7 +10,7 @@ import { Message } from '../message';
 import { Observable, Subject } from 'rxjs';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { debounceTime, takeUntil } from 'rxjs/operators';
-import { AccessService } from '../../../../../apps/codelab/src/app/modules/admin/services/access.service';
+import { AccessService } from '../../../../firebase-login/src/lib/access.service';
```

Comment:
I think you can just use here
import { AccessService } from '@codelab/firebase-login';